src/server/pps/persist/persist.proto

```diff
@@ -42,7 +43,7 @@ message JobState {
 message PipelineInfo {
   string pipeline_name = 1;
   pachyderm.pps.Transform transform = 2;
-  uint64 shards = 3;  // this is the degree of parallelism
+  uint64 parallelism = 3;  // this is the degree of parallelism
```

Comment:
Could you remove this comment? Now that it's called parallelism it seems like the comment isn't telling us much.